java/benchmarks/src/main/java/glide/benchmarks/clients/jedis/JedisClient.java

```diff
@@ -27,27 +33,36 @@ public String getName() {
 
     @Override
     public void connectToRedis(ConnectionSettings connectionSettings) {
-        if (connectionSettings.clusterMode) {
-            jedis =
+        isClusterMode = connectionSettings.clusterMode;
+        if (isClusterMode) {
+            jedisCluster =
                     new JedisCluster(
                             Set.of(new HostAndPort(connectionSettings.host, connectionSettings.port)),
                             DefaultJedisClientConfig.builder().ssl(connectionSettings.useSsl).build());
-        } else {
-            try (JedisPool pool =
-                    new JedisPool(
-                            connectionSettings.host, connectionSettings.port, connectionSettings.useSsl)) {
-                jedis = pool.getResource();
-            }
         }
+        jedisPool =
```

Comment:
why is the jedis pool always set, and not only when isClusterMode is false?

Reply:
Fixed in 2d0ec9d26200425c7b70906693ee9d077063fb82